src/clj_chrome_devtools/automation/launcher.clj

```diff
@@ -40,13 +40,14 @@
   (log/trace "Launching Chrome headless, binary: " binary-path
              ", remote debugging port: " remote-debugging-port
              ", options: " (pr-str options))
-  (let [args (remove nil?
-                     [binary-path
-                      (when (:headless? options) "--headless")
-                      (when (:no-sandbox? options) "--no-sandbox")
-                      "--disable-gpu"
-                      (str "--remote-debugging-port=" remote-debugging-port)
-                      (when-let [url (:url-to-open options)] (str url))])]
+  (let [args (->> [binary-path
+                   (when (:headless? options) "--headless")
+                   (when (:no-sandbox? options) "--no-sandbox")
+                   (when-not (:enable-gpu? options) "--disable-gpu")
+                   (str "--remote-debugging-port=" remote-debugging-port)
+                   (:args options)]
+                  flatten
```

Comment:
why flatten? I'd rather have concat or can `(:args options)` have deeply nested collections

Reply:
Out of habit I guess. Changed to concat.